spring-context/src/test/java/org/springframework/scheduling/annotation/AsyncExecutionTests.java

```diff
@@ -78,40 +79,35 @@ public void asyncMethods() throws Exception {
 		try {
 			asyncTest.returnSomething(0).get();
 			fail("Should have thrown ExecutionException");
-		}
-		catch (ExecutionException ex) {
+		} catch (ExecutionException ex) {
```

Comment:
Please undo all removal of newlines before `catch` statements.

See the following for details: https://github.com/spring-projects/spring-framework/wiki/Code-Style#braces